app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_with(model: [@project, task]) do |form| %>
+        
+    <div>
+        <%= form.label :name %>
+        <%= form.text_field :name %>
+    </div>
+    <div>
+        <%= form.label :title %>
+        <%= form.text_field :title %>
+    </div>
+    <div>
+        <%= form.label :description %>
+        <%= form.text_field :description %>
+    </div>
+    <div>
+        <%= form.label :deadline_at %>
+        <input type="date" name="task[deadline_at]" id="task[deadline_at]" value=<%= Time.now + 7.days %> />
```

Comment:
It is incorrect. In this implementation, the user can change the default deadline value